pom.xml

```diff
@@ -68,34 +68,12 @@
       <version>${kafka.version}</version>
     </dependency>
 
-    <dependency>
-      <groupId>org.apache.kafka</groupId>
-      <artifactId>kafka_2.12</artifactId>
-      <version>${kafka.version}</version>
-      <exclusions>
-        <exclusion>
-          <groupId>org.scala-lang</groupId>
-          <artifactId>scala-library</artifactId>
-        </exclusion>
-      </exclusions>
-    </dependency>
-
     <!-- Kafka requires these dependencies: declare this dependency to force vertx-kafka-client to use this one. These are the versions used by vert.x -->
     <dependency>
       <groupId>org.slf4j</groupId>
       <artifactId>slf4j-api</artifactId>
       <version>1.7.21</version>
     </dependency>
-    <dependency>
```

Comment:
have you checked that the kafka dependencies don't declare these removed dependencies ? what is kafka now using for logger by default ?

Reply:
@steven-aerts please take a look that removing this scala dependency is quite interesting

Reply:
@vietj @ppatierno The` kafka-client` is the pure java implementation of the kafka client protocol with minimal dependencies to be used as a library.  So you do not need to depend on the scala based kafka server implementation any more to communicate with kafka.  

The only relevant dependency which still needed to be fixed to prevent collisions was `slf4j-api`.

https://mvnrepository.com/artifact/org.apache.kafka/kafka-clients/2.3.0